examples/xgboost_simple.py

```diff
@@ -38,17 +38,20 @@ def objective(trial):
 
     param = {
         "silent": 1,
+        "seed": 1,
         "objective": "binary:logistic",
-        "booster": trial.suggest_categorical("booster", ["gbtree", "gblinear", "dart"]),
+        "booster": trial.suggest_categorical("booster", ["gbtree", "dart"]),
+        "min_child_weight": trial.suggest_loguniform("min_child_weight", 1e-3, 1e3),
```

Comment:
`1e3` seems quite large, even though it's in log scale. It usually ends up at around `0.3` when I run it locally and the [breast cancer dataset only contains around ~500 samples](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_breast_cancer.html). Is this number also based on your references?

Reply:
I agree. In [reference,](https://www.analyticsvidhya.com/blog/2016/03/complete-guide-parameter-tuning-xgboost-with-codes-python/), it is tuned as an integer and ends up to `6`. The range of the tuning seems to be `{1, 3, 5} -> {4, 5, 6} -> {6, 8, 10, 12}` (some stepwise tuning).